test/apps/test_init_mt.c

```diff
@@ -14,10 +14,22 @@
 #include <omp.h>
 #endif
 
+#include <time.h>
+#include <sys/time.h>
+#include <stdlib.h>
+
 
 int main(int argc, char **argv)
 {
-    int count = 0;
+    time_t timeout = (argc > 1) ? atol(argv[1]) : 60;
```

Comment:
no need this since this test should check for deadlock, not "too long" time

Reply:
> no need this since this test should check for deadlock, not "too long" time

removed